frontend/catalyst/device/qjit_device.py

```diff
@@ -304,12 +313,22 @@ def __init__(self, original_device):
 
         # Capability loading
         original_device_capabilities = get_device_capabilities(original_device)
+
+        # TODO: This is a temporary measure to ensure consistency of behaviour. Remove this
+        #       when customizable multi-pathway decomposition is implemented.
+        if hasattr(original_device, "_to_matrix_ops"):
+            _to_matrix_ops = (getattr(original_device, "_to_matrix_ops"),)
+            setattr(original_device_capabilities, "to_matrix_ops", _to_matrix_ops)
+            if _to_matrix_ops and "QubitUnitary" not in original_device_capabilities.operations:
+                raise CompileError(
+                    "The device that specifies to_matrix_ops must support QubitUnitary."
+                )
```

Comment:
Can you link a ticket or shortcut story id here with a little bit more context. I know this is a big story and it won't get solved in a single ticket. But it would be good to have a place to look at for more context later on.

Reply:
There's an entire epic for restructuring decompositions: https://app.shortcut.com/xanaduai/epic/74474?group_by=none&vc_group_by=day&ct_workflow=all&cf_workflow=500000005

Reply:
Also added a mention of the sc epic to the inline comment.